src/NuGet.Core/NuGet.Commands/RestoreCommand/RestoreCommand.cs

```diff
@@ -328,21 +330,11 @@ private static bool ValidateRestoreGraphs(IEnumerable<RestoreTargetGraph> graphs
                         // Log a summary with compatibility error counts
                         if (projectCount > 0)
                         {
-                            logger.LogError(
```

Comment:
The errors are logged when the issue is created which is what you see higher up in the file.

This part and the one below are basically information summary messages that are being logged as errors. There's no reason to log all of the issues and then also log another error saying there were X number of errors just logged.